articles/how-to-execute-linux-commands-in-python/index.md

```diff
@@ -0,0 +1,91 @@
+
+**Linux** is one of the most popular operating systems used by software developers and system administrators. It is open-source, free, customizable, and is very robust and adaptable making it an ideal choice for servers, [Virtual machines(VMs)](https://www.vmware.com/topics/glossary/content/virtual-machine), and an array of other use cases. Therefore, it is essential for anyone working in the tech industry to know how to work with Linux because it is used almost everywhere. *In this tutorial, we are going to look at how we can automate and run Linux commands in Python*. 
+
+### Table of contents
+[Prerequisites](#prerequisites)
```

Comment:
You need to add bullet points to this
It is rendering on the same line.

For example,

```markdown
- [Prerequisites](#prerequisites)
- [Introduction](#introduction)
- [Building an application to ping servers](#building-an-application-to-ping-servers)
- [Code](#code)
- [Conclusion](#conclusion)
```